From 6d5dda2ab05278d29faf79e22bfa0cf07052197e Mon Sep 17 00:00:00 2001
From: Vendula Poncova <vponcova@redhat.com>
Date: Wed, 9 Jan 2019 14:28:11 +0100
Subject: [PATCH] Move attributes from the Blivet class to Anaconda
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Organization: Invisible Things Lab
Cc: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>

Anaconda-specific attributes can be moved from the Blivet class in
Blivet to the InstallerStorage class in Anaconda.

Signed-off-by: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>
---
 pyanaconda/storage/osinstall.py | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/pyanaconda/storage/osinstall.py b/pyanaconda/storage/osinstall.py
index bfee5ccfb..4c341504c 100644
--- a/pyanaconda/storage/osinstall.py
+++ b/pyanaconda/storage/osinstall.py
@@ -1176,6 +1176,16 @@ class InstallerStorage(Blivet):
             :type ksdata: :class:`pykickstart.Handler`
         """
         super().__init__()
+        self.do_autopart = False
+        self.encrypted_autopart = False
+        self.encryption_cipher = None
+        self.escrow_certificates = {}
+
+        self.autopart_escrow_cert = None
+        self.autopart_add_backup_passphrase = False
+        self.autopart_requests = []
+
+        self._default_boot_fstype = None
 
         self.ksdata = ksdata
         self._bootloader = None
-- 
2.20.1

